concent_api/core/tests/utils.py

```diff
@@ -163,6 +163,13 @@ def _sign_message(self, golem_message, client_private_key = None):
             self.REQUESTOR_PRIVATE_KEY if client_private_key is None else client_private_key,
         )
 
+    def _generate_ethereum_signature(self, task_to_compute, requestor_ethereum_private_key=None):
```

Comment:
It is not described in https://github.com/Code-Poets/coding-style/issues/7, if type signatures should be also in testing utils. If it is no problem please add typing to this method. In this case types are pretty obvious, but lets keep it unified (as below :) )

Reply:
:+1: good point, done.